docs/BEMAN_STANDARD.md

```diff
@@ -5,8 +5,9 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # The Beman Standard
 
 This document specifies rules and recommendations for Beman project libraries.
-Its goal is to create consistency facilitating the evaluation of, and
-contribution to Beman libraries.
+Its goal is to create consistency facilitating the evaluation of, and contribution to Beman libraries.
```

Comment:
I think it is easier to follow a review when you break a line by the ending dot of the phrase. I did the changes in #80 as a model.

Let's say for example you have the following line:

`This is a phrase. This is a second line. This is a third line`.  When you modify a single word, the whole line will be marked as changed, allthough we meant to change only one word.

I find it odd to have line breaks at certain position, since it is quite difficult to impose a guideline about when somebody should break the line. And we also aim to have a linter workflow which verifies these changes.